sdk/communication/AzureCommunicationChat/Source/Token/PolicyTokenCredential.swift

```diff
@@ -50,7 +49,7 @@ public class CommunicationPolicyTokenCredential: TokenCredential {
      - scopes: A list of a scope strings for which to retrieve the token.
      - completionHandler: A completion handler which forwards the access token.
      */
-    public func token(forScopes _: [String], completionHandler: @escaping TokenCompletionHandler) {
+    public func token(forScopes _: [String] = [], completionHandler: @escaping TokenCompletionHandler) {
```

Comment:
Why the default is changed to empty here?

Reply:
This was a request from the arch review to have a default parameter of `[]`

Reply:
Yes, especially since it isn't used, it's silly to make customers supply something here, but needs to match the signature for the protocol